language/fixtures/super.rb

```diff
@@ -471,4 +471,26 @@ def foo(**)
     class C < A
     end
   end
+
+  module FromBasicObject
+    def __send__(name, *args, &block)
+      super
```

Comment:
So this calls `BasicObject#__send__`. This seems quite meta, would another method of BasicObject like `equal?` also work for this spec?